src/app/not-found.tsx

```diff
@@ -1,5 +1,14 @@
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <div className="mx-auto grid h-screen grid-cols-1 content-center">
+      <div className="flex justify-center">
+        <p className="font-monster text-4xl font-normal">
```

Comment:
```suggestion
        <p className="text-4xl font-normal">
```
montserrat is the default font, make sure your dev branch is up to date and then pull from dev